RxExample/RxExample/Examples/ImagePicker/ImagePickerController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  ImagePickerController.swift
+//  RxExample
+//
+//  Created by Segii Shulga on 1/5/16.
+//  Copyright © 2016 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+import UIKit
+#if !RX_NO_MODULE
+   import RxSwift
+   import RxCocoa
+#endif
+
+struct ImagePickerConfiguration {
+   let soruceType:UIImagePickerControllerSourceType
+   let allowsEditing:Bool
+}
+
+class ImagePickerController: ViewController {
+
+   @IBOutlet var imageView: UIImageView!
+   @IBOutlet var cameraButton: UIButton!
+   @IBOutlet var galleryButton: UIButton!
+   @IBOutlet var cropButton: UIButton!
```

Comment:
Yes, my bad, in my team we decided to use 3 spaces, and for this changes I forgot to change that in settings, Sorry :(
Will make changes